src/main/java/nicole/task/Task.java

```diff
@@ -0,0 +1,91 @@
+package nicole.task;
+
+import java.time.LocalDate;
+
+import nicole.nicoleexceptions.NicoleException;
+
+public class Task {
+    private boolean taskCompleted;
```

Comment:
Boolean variables should be named with a "is","has","was" prefix (as stated by the Java conventions in CS2103)